ui/component/viewers/videoViewer/internal/videojs-events.jsx

```diff
@@ -49,16 +50,28 @@ const VideoJsEvents = ({
   clearPosition: (string) => void,
   uri: string,
   doAnalyticsView: (string, number) => any,
+  doAnalyticsBuffer: (string, any) => void,
   claimRewards: () => void,
   playerServerRef: any,
   isLivestreamClaim: boolean,
 }) => {
+  function doTrackingBuffered(e: Event, data: any) {
+    const playerPoweredBy = isLivestreamClaim ? 'lvs' : playerServerRef.current;
+
+    data.playPoweredBy = playerPoweredBy;
+    data.isLivestream = isLivestreamClaim;
+    // $FlowFixMe
+    data.bitrateAsBitsPerSecond = this.vhs?.playlists?.media?.().attributes?.BANDWIDTH;
```

Comment:
[Minor]
<img width="369" alt="image" src="https://user-images.githubusercontent.com/64950861/168775765-bac5e8f3-6490-4bca-b324-7345db15c8bb.png">
These errors could be suppressed with `tech(true).vhs.playlists.media().attributes.BANDWIDTH`
(using `tech` as instructed, and passing `true` to say hide the "directly using tech" warning).